contracts/root/RootERC20Predicate.sol

```diff
@@ -74,6 +76,10 @@ contract RootERC20Predicate is Initializable, IRootERC20Predicate {
         _deposit(rootToken, msg.sender, amount);
     }
 
+    function depositNativeTo(address receiver) external payable {
```

Comment:
Add to IRootERC20Predicate and have documentation indicating this doc inherits from the interface

Reply:
I can clean up docs and more if we feel like this is the 'overall' right approach we want to take @drinkcoffee . ATM it's just PoC level code